File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>


int main(void)
{
    int size;
do
{
//Promte for user input
size = get_int("Height: ");
}
while(size < 1 || size > 8);

//Display pyramids
for (int i = 0; i < size; i++)
 {
     //Spaces, has to be > 1 so that the final row doesnt have a space
     for (int gap = size - i; gap > 1; gap--)
     {
            printf(" ");
     }
     //Hashtags
    for (int blocksl = 1; blocksl <= i + 1; blocksl++)
     {
        printf("#");
     }
    //Gap
        printf("  ");
     for (int blocksr = 1; blocksr <= i + 1; blocksr++)
     {
         printf("#");
     }
    //Need to start the new line (almost forgot)
        printf("\n");
    }
}

File: CS50/pset6/mario/more/mario.py
from cs50 import get_int
"""
Program prints a pyramid with 2 spaces in the middle,
like the one in mario
"""
# Do-while loop for the user input
while True:
    size = get_int("Height: ")
    if size > 0 and size < 9:
        break

#
for i in range(size):
    # space
    for gap in range(size - i, 1, -1):
        print(" ", end="")
    # Left hashtags
    for blocksl in range(i + 1):
        print("#", end="")
    # Middle Gap
    print("  ", end="")
    # The right hashtags
    for blocksr in range(i + 1):
        print("#", end="")
    # New line for next set
    print()